Sources/App/Views/PackageController/Package+ModelSupport.swift

```diff
@@ -28,7 +28,9 @@ extension Package {
             .first()
             .unwrap(or: Abort(.notFound))
     }
-    
+
+    // FIXME: inline into Analyzer.updateLatestVersions and use latest property instead
+    @available(*, deprecated)
     func defaultVersion() -> Version? {
```

Comment:
I’ll make this change in a follow-up PR

Reply:
Also note: we’ll need to deploy this PR here and let it run a full processing cycle on all records before I replace these methods with the shortcuts using the new field (or just live with the brief window of not showing the proper versions until everything’s caught up).

Reply:
Is it worth doing some kind of one-time manual verification after this processing has completed just to make sure that the new versions line up with the old? Is there a SQL query you have handy to select these three versions without the three fields that we could export as a CSV and compare against a similar query that runs from the tags after processing?

Might not be worth it, but it could be a good "belt and braces" test.

Reply:
Yes, I was going to run a check on staging. I've already checked that we are in fact updating all packages, i.e. we can be sure everything gets analysed within a couple of hours and there are no stale packages.

I wasn't going to do a 1:1 comparison, just a check that there are no packages with all versions having `latest = NULL`. That should suffice.

Reply:
OK, so we shouldn't push the deployment through to production until that's done then 👍

It's deploying to dev now, then we can let it do a full processing and inspect the results there. Sounds good.

Reply:
Yeah, it's probably best to pause twice: once to check the `latest` field is set and then, after deploying to prod, to wait and let the processing convert them all before we deploy the follow-up PR that uses the new field.